bsdlib/README.rst

```diff
@@ -87,6 +87,48 @@ The figure below shows a simplified BSD library architecture.
 
    BSD library architecture diagram
 
+
+Security tag model
+******************
+
+The modem on an nRF91 device is equipped with a full IPv4/IPv6 stack and a DTLS/TLS engine.
+To use the cryptographic functions in the modem, an application must provision the security assets to the modem before using them.
```

Comment:
What do we mean with "security assets"? In other documentation, we talk about [certificates](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf9160/aws_fota/README.html#cmdoption-arg-cloud-cert-sec-tag) or [credentials (keys or certificates)](https://infocenter.nordicsemi.com/topic/ug_link_monitor/UG/link_monitor/lm_certificate_manager.html), which I think is more specific than "assets".